src/content/whats-new/2024/09/whats-new-09-11-logs-in-context-in-kubernetes-deployments.md

```diff
@@ -12,17 +12,6 @@ To get started, open a Kubernetes deployment view and click the **Logs** tab. A
 
 In addition, we have added the **Mini Overview** widget to the Kubernetes deployment summary page, allowing you to view the total log count and the error log count associated with your Kubernetes deployment. This widget contains the **See logs** button, which opens up the logs in context view for that Kubernetes deployment.
 
-<img
-    title="A screenshot showing logs in context view for a kubernetes deployment"
-    alt="A screenshot showing logs in context view for a kubernetes deployment"
-    src={/images/WhatsNewLogsInContextKubernetesDeployment.webp}
-/>
-<figcaption>A screenshot showing logs in context view for a kubernetes deployment.</figcaption>
-
-<img
-    title="A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page"
-    alt="A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page"
-    src={/images/WhatsNewMiniOverviewWidgetInKubernetesDeploymentSummaryPage.webp}
-/>
-<figcaption>A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page.</figcaption>
+![Logs in context view](/images/WhatsNewLogsInContextKubernetesDeployment.webp "A screenshot showing logs in context view for a kubernetes deployment")
 
+![LogsMiniOverviewWidget](/images/WhatsNewMiniOverviewWidgetInKubernetesDeploymentSummaryPage.webp "A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page")]
```

Comment:
@jeff-colucci There's a stray square bracket at the very end of this line that needs to be deleted, once you do that, and the builds finish, feel free to merge it in

Reply:
```suggestion
![LogsMiniOverviewWidget](/images/WhatsNewMiniOverviewWidgetInKubernetesDeploymentSummaryPage.webp "A screenshot showing the LogsMiniOverviewWidget in the kubernetes deployment summary page")
```